.dockerignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+**/.terraform/**
+.git
```

Comment:
The build was pulling in all the .terraform files, which made the build huge. Like 10 GB